Sources/ViewInspector/SwiftUI/TreeView.swift

```diff
@@ -9,7 +9,15 @@ internal extension ViewType {
 extension ViewType.TreeView: SingleViewContent {
 
     static func child(_ content: Content) throws -> Content {
-        let view = try Inspector.attribute(path: "root|content", value: content.view)
+        let view: Any = try {
+            guard let rootContent = try? Inspector.attribute(path: "root|content", value: content.view) else {
+                // Try to return the `contents` property directly if the tree root does not contain a View.
+                // (e.g. Layout, _VariadicView_MultiViewRoot, _VariadicView_UnaryViewRoot)
+                return try Inspector.attribute(path: "content", value: content.view)
+            }
```

Comment:
It could be nice if we had a comment indicating that one version exists for older SwiftUI versions (with the version number). Then the code can be cleaned up if the library stops supporting older SwiftUI versions.

Reply:
True, but sometimes it's hard to tell at which exact os version the change took place - you'd need to do a "bisect" on a wide range of iOS versions, and have them all installed